azure-mgmt-graph-rbac/src/main/java/com/microsoft/azure/management/graphrbac/implementation/RoleDefinitionImpl.java

```diff
@@ -74,18 +76,18 @@ public String type() {
     }
 
     @Override
-    public List<PermissionInner> permissions() {
+    public Set<PermissionInner> permissions() {
```

Comment:
since PermissionInner is an object, i don't think we'd want a Set here, but a Collection. Sets work best only for Strings.